### PR TITLE
Syndicate Item: Pseudocider - Feign your death, with security none the wiser!

### PR DESCRIPTION
**Pull request by ynot01**
# Document the changes in your pull request

Adds the Pseudocider

<details>
  <summary>Pseudocide</summary>

image

</details>

A small item for 6TC that can be turned on in-hand or activated from the pocket using an action button. Once toggled on (indicated by an open face watch), the next time you take brute, burn, or stamina damage, you will be made invisible and a fake body will be left dead or stunned in your position. You will also receive a 75% damage **slowdown** resistance and will be unable to click on or use pretty much anything, but can walk into doors (and bullets).

The body left behind is a real mob that copies your damage, equipment (clothing), implants, and will not appear catatonic or SSD when examined.

After 7 seconds, you will become visible, and the body will be deleted, alerting everyone around it that it was a fake copy.

Has a cooldown of 20 seconds that starts when the cloak ends.

See below gifs for gameplay

# Why is this good for the game?
Biome wanted it

Also it's cool and stuff like fool security haha hehe

# Testing

<details>
  <summary>Gameplay</summary>

The below is me examining myself and the fake body
image

</details>

# Spriting
Sprites by thegoldencat413

<details>
  <summary>Sprites</summary>

image
image

</details>

# Wiki Documentation
6TC. Cloak lasts 7 seconds and cannot be cancelled. Recharge time is 20 seconds, starting from the decloak.

# Changelog

:cl:  ynot01, thegoldencat413
rscadd: Added the pseudocider to the stealthy tools uplink section for 6TC. Activate it and the next time you take damage, you will feign your death or stun! 
/:cl:

### CODE REVIEW

**Comment by ToasterBiome** [comment]
me like

**Comment by maxion12345** [comment]
can this be countered by using fire

**Comment by ynot01** [comment]
> can this be countered by using fire 

i dont think so but the dead ringer removes afterburn anyways

**Comment by azzzertyy** [comment]
, equipment (clothing),
i am SURE drake will NOT abuse this.

**Comment by ynot01** [comment]
> , equipment (clothing), i am SURE drake will NOT abuse this.

Elaborate and maybe I can prevent this, but I have covered every exploit that I could think of

**Comment by ynot01** [comment]
to-do: add race features

**Comment by bruhlookatthisdood** [comment]
i am ze spy

File: code/game/objects/items/devices/pseudocider.dm [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+	if(HAS_TRAIT(user, TRAIT_NOINTERACT))
+		return
+	if(!COOLDOWN_FINISHED(src, fake_death_timer) && !active)
+		to_chat(user, span_notice("\The [src] refuses to open! Wait [COOLDOWN_TIMELEFT(src, fake_death_timer)/10] more seconds!"))
+		return
+	active = !active
+	if(active)
+		icon_state = "pocketwatch-open"
+	else
+		icon_state = "pocketwatch-closed"
```
Comment (JohnFulpWillard) [inline code/game/objects/items/devices/pseudocider.dm]: This should be in update_icon and should use ``base_icon_state``

File: code/game/objects/items/devices/pseudocider.dm [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+
+	COOLDOWN_START(src, fake_death_timer, fake_death_cooldown) // sanity
+
+	fake_corpse = new copied_mob.type
+	if(copied_mob.dna?.species)
+		INVOKE_ASYNC(fake_corpse, TYPE_PROC_REF(/mob,set_species), new copied_mob.dna.species.type)
+		fake_corpse.dna.features = LAZYCOPY(copied_mob.dna.features)
+
+	// Set up the mob to be identical
+	if(ishuman(fake_corpse) && ishuman(copied_mob))
```
Comment (JohnFulpWillard) [inline code/game/objects/items/devices/pseudocider.dm]: Do we not have any helpers to do this stuff?
If not, can you make these all helpers?

File: code/modules/mob/living/carbon/human/examine.dm [hunk 3]
```diff
@@ -407,7 +407,7 @@
 		if(getorgan(/obj/item/organ/brain))
 			if(!key)
 				msg += "[span_deadsay("[t_He] [t_is] totally catatonic. The stresses of life in deep-space must have been too much for [t_him]. Any recovery is unlikely.")]\n"
-			else if(!client)
+			else if(!client && !fake_client)
 				msg += "[t_He] [t_has] a blank, absent-minded stare and appears completely unresponsive to anything. [t_He] may snap out of it soon.\n"
```
Comment (JohnFulpWillard) [inline code/modules/mob/living/carbon/human/examine.dm:411]: Isn't the body supposed to be dead anyways? Reviving it should be a tell no?

**Comment by thegoldencat413** [comment]
sprites kinda suck

File: code/modules/mob/living/carbon/human/examine.dm (hunk 3 above)
Comment (ynot01) [reply code/modules/mob/living/carbon/human/examine.dm:411]: Body does not die if damage taken is stamina damage

**Comment by ynot01** [comment]
Tested locally

**Review by JohnFulpWillard — APPROVED** [review]

**Comment by RG4ORDR** [comment]
How would this work with say - the cloaker belt?

**Comment by ynot01** [comment]
> How would this work with say - the cloaker belt?

both items modify alpha so it would conflict a bit but as cloaker belt sets your alpha often it would essentially take priority

**Review by cuackles — APPROVED** [review]

**Pull request merged**